src/components/pick-list/shared-list-render.tsx

```diff
@@ -31,6 +31,9 @@ export const List: FunctionalComponent<{ props: ListProps } & DOMAttributes<any>
   return (
     <Host aria-busy={loading.toString()} role="menu" {...rest}>
       <section>
+        <span aria-live="assertive" class="assistive-text">
+          {" "}
```

Comment:
Is this displayed?

Reply:
nope. this isn't displayed. it's just an assistive text element which has no width or height

Reply:
Will remove the empty string.